src/erlazure_insights.erl

```diff
@@ -73,9 +73,12 @@ http_request(SubscriptionId, Filter, Select, Config) ->
     URL = "https://management.azure.com/subscriptions/" ++
     SubscriptionId ++
     "/providers/microsoft.insights/eventtypes/management/values?" ++ "api-version=2014-04-01" ++ 
-    "&%24filter=" ++ Filter ++
-    "&%24select=" ++ Select,
-    request_api(URL, Config).
+    "&%24filter=" ++ Filter,
```

Comment:
The reason we can't use http_uri:encode here is that it does not encode $ -> %24